controllers/lrp_convergence_controller.go

```diff
@@ -179,7 +179,7 @@ func (h *LRPConvergenceController) ConvergeLRPs(ctx context.Context) {
 	for _, lrpKey := range convergenceResult.UnstartedLRPKeys {
 		dereferencedKey := *lrpKey
 		works = append(works, func() {
-			before, after, err := h.lrpDB.UnclaimActualLRP(ctx, logger, dereferencedKey.Key)
+			before, after, err := h.lrpDB.UnclaimActualLRP(ctx, logger, true, dereferencedKey.Key)
```

Comment:
Please add a test for this. There should be more tests to fix for this function signature change.